.github/workflows/push-tests.yml

```diff
@@ -1,12 +1,12 @@
 name: Push Tests
 
 on:
-  pull_request:
-    branches:
-      - "*"
   push:
     branches:
       - "main"
+      - "prod"
```

Comment:
do we want push tests to run in prod? we only promote if push tests pass on that commit in main, so it seems redundant

Reply:
I included this originally because we were making pull requests from main->prod which would have triggered the push test. So in order to keep this running as it was before I should add 
```
on:
   pull-requests:
      branches:
        - "prod"
```
However the new workflow for deploying prod uses a GHA called via workflow which would set off the pull-request trigger. Adding "prod" to push would make sure the push-tests run on prod at least one. But as you said this is probably redundant. I'm happy to remove if that is the consensus.

Reply:
my vote is to remove, but I don't mind if you want to ask around and get a consensus. I think a lot of people are out today, though